### ISSUE TITLE
[FEATURE] Add sorting by "Modified date"

### ISSUE THREAD

**Issue by stelpstraATeur** [UtrechtUniversity/yoda-ansible#343]
#### Is your feature request related to a problem? Please describe.
When hundreds or thousands of files are present and the file names are not numbered in a chronological order, it is very difficult to find the latest file(s) or analyse the sequence of files as the default sort method is the Name column. An export of the metadata of files containing also the dates is not available either, the "Show checksum" export only lists the name of the files and its checksums.

#### Describe the solution you'd like
To list files in chronological order of creation or modification and/or find files of a specific creation / modification date and time, add functionality to sort on the date and time of the files.

#### Describe alternatives you've considered
...

#### Additional context
...

**Comment by Danny-dK** [comment]
Note that Yoda sets the modification date to the date and time of upload (quick tested this with data uploaded through the webportal and as a windows network drive). This could mean that currently the modification date may be less relevant and could depend on how the researcher is using Yoda. Perhaps Yoda could retain the modification date of the originally uploaded file? (but that might also not reflect the actual modification date, better to use best RDM practices as in setting date within the filename itself)

**Comment by stelpstraATeur** [comment]
File dates are just the most basic file management data, in many systems we find dates of creation and modification. In the Yoda context a timestamp of when it was uploaded could also be very useful. Sorting on any of those dates seems to be quite useful too.

**Comment by Danny-dK** [comment]
I agree it is the most basic information available, but may be misleading especially currently in Yoda. For example, if the files uploaded do not have numeric versioning and dates in the filename:
- file_new.csv
- file_newest.csv
- file_newfinal.csv (lets say this is the actual final one)

Then if a user upload 'newest' instead of the actual final version, that won't be visible in Yoda in the dates shown there, it will just show upload date (and how useful is that date). This example can become more of a difficult situation when the user has multiple folder with files containing similar names between folders. If a wrong file from a folder is chosen that is of a different earlier modification date, that will also not be visible once in Yoda. The usage of the file metadata 'date' is to find when something was last changed in a file. To deal with differences in how applications deal with these dates, RDM practices advise to add date and versioning in filenames (hence responsibility of users). 

But if Yoda is able to retain modification date for file metadata upload and display that, I can then imagine it to be useful to sort on dates. I would also imagine to change the current heading in Yoda from 'modified date' to 'upload data' as the term 'modified date' may be misleading in its meaning.

**Comment by RobvanSchip** [comment]
Thank you for the feature request. This is planned for version 2.0 and we have registered it in our product backlog as ticket nr. [YDA-5805](https://utrechtuniversity.atlassian.net/browse/YDA-5805)
